core/src/empty/empty.module.css

```diff
@@ -0,0 +1,14 @@
+.container {
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    font-family: Inter;
+}
+
+.errorText {
+    font-style: normal;
+    font-weight: normal;
+    font-size: 14px;
+    line-height: 20px;
```

Comment:
These are all unnecessary styling. They are already so either by default (line 9 and 10) or inherited (line 11 and 12)